src/main/java/core/basesyntax/report/convertdata/DataConvertorImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.report.convertdata;
+
+import core.basesyntax.model.FruitOperation;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConvertorImpl implements DataConvertor {
+    private static final String CSV_SEPARATOR = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_NAME_INDEX = 1;
+    private static final int REPORT_START_POSITION = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitOperation> convertToTransaction(List<String> fruitInfoList) {
+        List<FruitOperation> fruitTransactions = new ArrayList<>();
+        for (int i = REPORT_START_POSITION; i < fruitInfoList.size(); i++) {
```

Comment:
Avoid hardcoding values. The 'REPORT_START_POSITION' constant is used to skip the first line, assuming it's a header. It would be better to make this behavior explicit in the code or configurable, in case the input format changes.